lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1247,6 +1247,12 @@ def test_get_context(self):
         rendered_context = self.descriptor.get_context()
         self.assertListEqual(rendered_context['tabs'], correct_tabs)
 
+        # Assert that the Video ID field is present in basic tab metadata context.
```

Comment:
I see this test is for video_mongo, do we need to check for split as well or at-least we need to be sure